tck/src/main/java/org/eclipse/microprofile/lra/tck/TckTests.java

```diff
@@ -530,16 +530,16 @@ private void joinWithOneResource(String methodName, boolean close, String resour
             lraClient.closeLRA(lra);
             lraTestService.waitForCallbacks(lra);
             
-            assertTrue(methodName + ": resource should have completed once with no compensations",
```

Comment:
Can we keep this text since it provides more information that the changed text

Reply:
what do you mean? Now it doesn't have to complete only *once* so this text is no longer valid.

Reply:
It does need to complete so the text still applies.

Reply:
Ah I see you have added the deleted line back in again so I shall resolve this conversation.